lib/utils.js

```diff
@@ -208,4 +208,105 @@ exports.createNullProtoObjWherePossible = (function () {
   };
 })();
 
+// Embedded argument parser from https://github.com/jakejs/jake/blob/master/lib/parseargs.js
+// See original source for doc.
+// Fixes #510 and allows removing `jake` as a prod dependency.
+exports.parseArgs = function parseArgs(programOpts, programArgs) {
+  let isOpt = function (arg) {
+    return arg.indexOf('-') === 0;
+  };
+  let removeOptPrefix = function (opt) {
+    return opt.replace(/^--/, '').replace(/^-/, '');
```

Comment:
This may not matter, but you could just use one `replace` call by using the regex of `/^--?/` as that means that the second hyphen is optional.

Reply:
It also sorta sounds like this was just copy/pasted from jake, so I also understand this wasn't technically the op's code.